apps/web/src/server/applications/create/applicant/__tests__/applications-create-applicant.test.js

```diff
@@ -170,4 +170,51 @@ describe('applications create applicant', () => {
 			});
 		});
 	});
+
+	describe('Applicant telephone number', () => {
+		const baseUrl = '/applications-service/create-new-case/123/applicant-telephone-number';
+
+		describe('GET /applicant-telephone-number', () => {
+			it('should render the page if there is data in the session', async () => {
+				await request
+					.post('/applications-service/create-new-case/123/applicant-information-types')
+					.send({
+						selectedApplicantInfoTypes: ['applicant-telephone-number']
+					});
+
+				const response = await request.get(baseUrl);
+				const element = parseHtml(response.text);
+
+				expect(element.innerHTML).toMatchSnapshot();
+				expect(element.innerHTML).toContain('Save and continue');
+			});
+
+			it('should show not render the page if there is no session data', async () => {
```

Comment:
small spelling error

Reply:
updated wording and in all 4 previous tests.